apps/mdn/mdn-aws/infra/mdn-cdn/mdn_cdn.tf

```diff
@@ -25,26 +25,13 @@ module "mdn-primary-cloudfront-stage" {
 
 module "mdn-primary-cloudfront-prod" {
    source = "./cloudfront_primary"
-   acm_cert_arn = "arn:aws:acm:us-east-1:236517346949:certificate/68395c25-b5d4-4875-b85a-8c89b80c4e01"
-   aliases = ["developer.mozilla.org", "cdn.mdn.mozilla.net"]
+   acm_cert_arn = "arn:aws:acm:us-east-1:236517346949:certificate/144c40ad-1a60-4865-a252-58ff23961787"
```

Comment:
just dropping a note here to confirm that this is the us-east-1 cert _without_ `cdn.mdn.mozilla.net`